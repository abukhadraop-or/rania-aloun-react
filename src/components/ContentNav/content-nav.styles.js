import styled from 'styled-components';

export const NavBar = styled.ul`
  background-color: white;
  color: #98a69b;
  font-size: 0.938rem;
  list-style-type: none;
  margin: 0 0 0.53rem;
  padding: 0;
`;

export const NavElement = styled.li`
  border-bottom: 0.13rem solid #5cb85c;
  color: #5cb85c;
  cursor: pointer;
  display: inline;
  margin: 0 10px 0;
  margin: 0;
  padding: 0.5rem 1rem;
`;
