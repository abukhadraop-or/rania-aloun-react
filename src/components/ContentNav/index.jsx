import React from 'react';
import { NavBar, NavElement } from './content-nav.styles';

/**
 * Component holding the navigation bar on top of articles.
 *
 * @return {JSX.Element} Navigation bar holding the global feed.
 */
function ContentNav() {
  return (
    <NavBar>
      <NavElement>Global Feed</NavElement>
    </NavBar>
  );
}

export default ContentNav;
