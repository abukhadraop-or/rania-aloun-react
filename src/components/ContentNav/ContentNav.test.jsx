import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ContentNav from '.';

describe('<ContentNav />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ContentNav />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
