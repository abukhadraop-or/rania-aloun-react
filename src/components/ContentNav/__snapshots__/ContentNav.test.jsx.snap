// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentNav /> matches snapshot 1`] = `
<styled.ul>
  <styled.li>
    Global Feed
  </styled.li>
</styled.ul>
`;
